bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -156,54 +160,56 @@ def load_missed_found_injections(hdf_file, ifos, snr_threshold, bank_file,
                 found_data['sigmasq_%s' % ifo][:] = 1.0
         antenna = Detector(ifo)
         f_resp[ifo] = ppu.get_antenna_responses(antenna, found_data['ra'],
-                                            found_data['dec'],
-                                            found_data['tc'])
+                                                found_data['dec'],
+                                                found_data['tc'])
 
     inj_sigma_mult = \
         np.asarray([f_resp[ifo] *
-                       found_data['sigmasq_%s' % ifo] for ifo in ifos])
+                   found_data['sigmasq_%s' % ifo] for ifo in ifos])
     inj_sigma_tot = np.sum(inj_sigma_mult, axis=0)
     for ifo in ifos:
         found_data['inj_sigma_mean_%s' % ifo] = np.mean(
             found_data['sigmasq_%s' % ifo] * f_resp[ifo] / inj_sigma_tot)
     # Close the hdf file
     inj_data.close()
-    
+
     return found_data, missed_data
 
+
 # =============================================================================
 # Main script starts here
 # =============================================================================
 parser = ppu.pygrb_initialize_plot_parser(description=__doc__,
                                           version=__version__)
 parser.add_argument("-F", "--offsource-file", action="store", required=True,
                     default=None, help="Location of off-source trigger file")
-# As opposed to offsource-file and trig-file, this only contains onsource
 parser.add_argument("--onsource-file", action="store", default=None,
                     help="Location of on-source trigger file.")
-ppu.pygrb_add_injmc_opts(parser)
+parser.add_argument("--found-missed-file", action="store", default=None,
```

Comment:
I think there is no need to explicitly use `default=None` here and below. By default, there is no default :)